src/components/Copy/Copy.tsx

```diff
@@ -1,41 +1,48 @@
-import React, { useState } from 'react'
+import React, { FC, useState } from 'react'
 import { IconButton } from '@mui/material'
 import Tooltip from '../Tooltip'
 import copy from 'copy-to-clipboard'
 
 import { Copy as CopyIcon, CheckMark } from '../../svgs'
 
 import './copy.scss'
+import clsx from 'clsx'
 
 interface CopyProps {
   text: string
   extraClass?: string
+  CopyComponent?: FC<{ onClick: (e: React.MouseEvent) => void }>
+  copyText?: string
 }
 
-function Copy({ text, extraClass }: CopyProps) {
+function Copy(props: CopyProps) {
+  const { text, extraClass, CopyComponent, copyText } = props
+
   const [isCopyPress, setCopyPress] = useState(false)
 
-  function onClick() {
+  function onClick(e: React.MouseEvent) {
+    e.stopPropagation()
+
     copy(text)
     setCopyPress(true)
     setTimeout(() => setCopyPress(false), 2000)
   }
 
   return (
-    <Tooltip data={isCopyPress ? 'Copied!' : 'Copy'}>
-      <span className={extraClass}>
-        <IconButton
-          className='copy'
-          onClick={onClick}
-          disabled={isCopyPress}
-        >
-          {!isCopyPress ? (
-            <CopyIcon />
+    <Tooltip data={isCopyPress ? 'Copied!' : copyText ?? 'Copy'}>
```

Comment:
let's add default value ('Copy') to `copyText` in line 19, so that here it'll be `<Tooltip data={isCopyPress ? 'Copied!' : copyText}>`